src/engine/effects/message.h

```diff
@@ -125,9 +130,13 @@ struct EffectsRequest {
     // Message-specific, non-POD values that can't be part of the above union.
     ////////////////////////////////////////////////////////////////////////////
 
-    // Used by ENABLE_EFFECT_CHAIN_FOR_CHANNEL and DISABLE_EFFECT_CHAIN_FOR_CHANNEL.
+    // Used by ENABLE_EFFECT_CHAIN_FOR_INPUT_CHANNEL and
+    // DISABLE_EFFECT_CHAIN_FOR_INPUT_CHANNEL.
     ChannelHandle channel;
```

Comment:
Can't this become a pointer as well, and go the the union? Or a part of the pStatesForEffectsInChain?

Reply:
Ping!

Reply:
Ping!